colectica_api/api.py

```diff
@@ -764,6 +764,53 @@ def get_a_set_lasted(self, AgencyId, Identifier):
             if response.json() != []:
                 return response.json()
 
+    def update_state(
+        self,
+        items,
+        *,
+        Version=None,
+        State=False,
+        ApplyToAllVersions=True
+    ):
+        """Updates the Deprecated State of a set of items
+        Args:
```

Comment:
full stop after `items.`

I think a blank line between the first line and `Args:`.